scripts/quick-run/merge-devnets/docker-compose.geth.yml

```diff
@@ -12,10 +12,9 @@ services:
   geth:
     image: parithoshj/geth:merge-d99ac5a
     container_name: geth
-    ports:
```

Comment:
Definitely simpler to use `host` network mode here rather than listing all ports (so uber nit/probably outside the design here) but wondering what the thought is with something like: https://github.com/0x0I/container-file-geth/blob/e9695eea0fb05b67427bdc4c129ec665c423a613/compose/docker-compose.yml#L22 to potentially avoid local port conflicts and enable port customization.

Could be not worth having to think about tho the defaults basically yield the same result as host networking and each envvar can also just be set within the operator's runtime env (e.g. via something like `export rpc_port=8545`) or either set within a .env file processed by default or a custom file passed with `docker-compose up` provided alternate host parts are desired.